packages/studio/src/components/PropEditor.tsx

```diff
@@ -41,9 +41,9 @@ export default function PropEditor({
   const labelTooltipId = `${uniqueId}-label`;
 
   return (
-    <div className="flex items-center mb-2 text-sm">
-      {renderBranchUI(isNested, "pb-10")}
-      <label>
+    <div className="flex items-center mb-2 text-sm grow">
+      {isNested && renderBranchUI("pb-10")}
+      <label className="flex flex-col relative grow">
```

Comment:
I think some of this styling is causing the prop inputs to be clickable beyond the visible input I'd expect. Also, the undefined menu placing is a little off

https://github.com/yext/studio/assets/88398086/a4b9c258-a2c6-424c-890a-ba6405b4efc2

Reply:
Looks like clicking on the prop name still counts as clicking on the input, which feels a little weird for the color picker. But I think it's ok if it's difficult/messier to change it back.

The undefined menu placing still needs to be tweaked though
<img width="297" alt="Screen Shot 2023-08-28 at 1 50 29 PM" src="https://github.com/yext/studio/assets/88398086/83e0bf47-8333-41c4-a188-6246252a3b7c">

Reply:
Oh you're right, I hadn't noticed that before! It's just the undefined menu then that needs to be adjusted